packages/xo-server/src/api/vif.js

```diff
@@ -112,6 +117,7 @@ export async function set({
     ipv4Allowed: allowedIpv4Addresses,
     ipv6Allowed: allowedIpv6Addresses,
     rateLimit,
+    txChecksumming,
```

Comment:
You already create the VIF with it.

Reply:
There is a return. See https://github.com/vatesfr/xen-orchestra/pull/5182/files#diff-9fda1acd3467048cd34b35f0597c8053R107